src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Lightup/AllowsConstraintClauseSyntaxWrapper.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax;
+
+internal readonly struct AllowsConstraintClauseSyntaxWrapper
```

Comment:
This is a block as well.  Our features need to be able to use the latest syntax trees which expose the latest features directly.